src/palace/manager/api/admin/controller/library_settings.py

```diff
@@ -32,7 +32,9 @@
 from palace.manager.sqlalchemy.model.announcements import (
     SETTING_NAME as ANNOUNCEMENT_SETTING_NAME,
 )
-from palace.manager.sqlalchemy.model.announcements import Announcement
+from palace.manager.sqlalchemy.model.announcements import (
+    Announcement,
+)
```

Comment:
I couple of these import changes are a little weird, but it appears to be related to a change isort made in handling `as` imports on the previous line. Even though is a bit weird, I don't think its a problem.

Reply:
Because of these kind of weird imports, I tried enabling the `combine_as_imports` isort option in our config. 

The automatic results are shown in https://github.com/ThePalaceProject/circulation/pull/2411/commits/87602eb9a5606ad9f6220b709d4cd4fafd1462db. I think I like how things are sorted better with this option enabled.

Reply:
I prefer the `combine_as_imports` behavior.